cmd/podman/root.go

```diff
@@ -128,6 +128,16 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	for _, env := range cfg.Engine.Env {
+		splitEnv := strings.SplitN(env, "=", 2)
+		if len(splitEnv) != 2 {
+			return fmt.Errorf("invalid environment variable for engine %s, valid configuration is KEY=value pair", env)
+		}
+		if err := os.Setenv(splitEnv[0], splitEnv[0]); err != nil {
```

Comment:
should this be ` os.Setenv(splitEnv[0], splitEnv[1])` ?

Reply:
yes. fixed....